Gemfile.lock

```diff
@@ -74,7 +74,7 @@ GIT
 
 GIT
   remote: https://github.com/department-of-veterans-affairs/vets-json-schema
-  revision: 897ad81be907fa6bd3d997f20ff1e0a912d436cd
+  revision: 005c292c915aaf1df1f702ab0c44614ec6789df6
```

Comment:
Is this right? The version and revision were bumped an hour ago in https://github.com/department-of-veterans-affairs/vets-api/pull/21509.

Reply:
@kjsuarez and I reviewed the conflict over Slack and think that pushing this one through now will result in the most recent version of `vets-json-schema` being on `vets-api` (based on schema commit history, 005c292c915aaf1df1f702ab0c44614ec6789df6 was the second of the two to go in). They mistakenly have the same version number but this has some additional content

Reply:
ah, thank you for figuring this out together! I will approve this, then.